api/Dockerfile

```
@@ -13,12 +13,16 @@ FROM base AS runner
-# CVE-2026-6732 does not apply here; this mirrors deploy/Dockerfile's pattern.
-# APK_CACHEBUST is fed a per-build value so the buildcache for this layer is
-# invalidated each build and apk upgrade re-fetches current Alpine secfixes.
+# CVE-2026-31789 and CVE-2026-45447). The oven/bun:1.3.10-alpine base does not
+# ship libxml2, so CVE-2026-6732 does not apply here; this mirrors
+# deploy/Dockerfile's pattern. APK_CACHEBUST is fed a per-build value so the
+# buildcache for this layer is invalidated each build and apk upgrade re-fetches
+# current Alpine secfixes. The openssl libs are named explicitly so the fix for
+# the tracked CVE-2026-45447 alerts is force-applied.
 ARG APK_CACHEBUST=0
-RUN echo "apk-cachebust=${APK_CACHEBUST}" && apk upgrade --no-cache
+RUN echo "apk-cachebust=${APK_CACHEBUST}" && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)